lxd/seccomp/seccomp.go

```diff
@@ -1249,6 +1249,15 @@ func mountFlagsToOpts(flags C.ulong) string {
 	return opts
 }
 
+// mountHandleHugetlbfsArgs adds user namespace root uid and gid to the
+// hugetlbfs mount options to make it useable in unprivileged containers.
+func (s *Server) mountHandleHugetlbfsArgs(args *MountArgs, nsuid int64, nsgid int64) {
+	if args.fstype == "hugetlbfs" && args.data == "" {
+		args.data = fmt.Sprintf("uid=%d,gid=%d", nsuid, nsgid)
+		args.shift = false
+	}
```

Comment:
This only handles the case where the user didn't specify a uid/gid.

If `uid` or `gid` are specified by the user, we should parse them and shift them using the instance map.

Reply:
done

Reply:
Somehow missed that.